.travis.yml

```diff
@@ -7,5 +7,6 @@ rust:
 
 script:
   - if [ $TRAVIS_RUST_VERSION = nightly ]; then cargo build --verbose --manifest-path lpc11uxx/Cargo.toml; fi
+  - if [ $TRAVIS_RUST_VERSION = nightly ]; then cargo build --verbose --manifest-path lpc11u6x/Cargo.toml; fi
```

Comment:
Is this copy-pasted from the line above, or does the crate really only build on nightly?

Reply:
It's copy pasted, I assumed svd2rust requires nightly.

Reply:
It seems to build on stable.

Reply:
Cool! Please change it to built it unconditionally then, like the one's below it.

Reply:
Whoops, it doesn't build, it just generates the code successfully (but it doesn't compile).

Reply:
Understood. Thanks for looking into it.